drivers/windows/file_access_windows.cpp

```diff
@@ -559,11 +559,10 @@ uint64_t FileAccessWindows::_get_modified_time(const String &p_file) {
 		return 0;
 	}
 
-	String file = p_file;
+	String file = fix_path(p_file);
 	if (file.ends_with("/") && file != "/") {
```

Comment:
```suggestion
	String file = fix_path(p_file);
	if (file.ends_with("\\") && file != "\\") {
```

Since it's a check after `fix_path` it is Windows style path and using `\\` not `/`. The issue seems to be caused by calling `_get_modified_time("res://")` specifically.